Libs/Gst/Gst.csproj

```diff
@@ -5,5 +5,7 @@
 
   <PropertyGroup>
     <PackageId>Gir.CS.Gst</PackageId>
+    <LangVersion>9.0</LangVersion>
```

Comment:
`LangVersion` should not be needed as `Directory.Build.props` defines .NET 5 which uses C# 9 by default

Reply:
Fixed